src/sagemaker/image_uri_config/neo-mxnet.json

```diff
@@ -2,20 +2,20 @@
     "processors": ["cpu", "gpu"],
     "scope": ["inference"],
     "version_aliases": {
-        "0.12.1": "1.5",
-        "1.0.0": "1.5",
-        "1.1.0": "1.5",
-        "1.2": "1.5",
-        "1.2.0": "1.5",
-        "1.2.1": "1.5",
-        "1.3": "1.5",
-        "1.3.0": "1.5",
-        "1.4": "1.5",
-        "1.4.0": "1.5",
-        "1.4.1": "1.5"
+        "0.12.1": "1.7",
```

Comment:
Now all versions will map to 1.7?

Reply:
yeah. the older version has different repo now. If you compile a model with neo service now. The only container you can use to serve the model is the 1.7 one.

Reply:
🤕

Reply:
Thats right, Neo doesnt support compiling with different versions of any Framework (MX, TF, or PT) yet. Whatever version of the framework used to train the model, only the one version in Neo (i.e MX-1.7, PT-1.4, TF-1.15) will be used to load that model. backward compatibility is assumed